### PR TITLE
ZCS-7937: (ZXL-158) Jetty configuration for zimbra-x ui on legacy server

### PR DESCRIPTION
**Pull request by yashrajpchavda**
updated the jetty configuration for zimbrax on legacy

### CODE REVIEW

File: conf/jetty/jetty.xml.production [hunk 1]
```diff
@@ -614,7 +614,73 @@
...
+          </Arg>
+        </Call>
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">.*(.chunk.*)</Set>
+              <Set name="name">Cache-Control</Set>
+              <Set name="value">Max-Age=31536000,public</Set>
+              <Set name="terminating">true</Set>
```
Comment (silentsakky) [inline conf/jetty/jetty.xml.production]: check if terminating is required or not
check for all rewrites

**Review by silentsakky — APPROVED** [review]

File: conf/jetty/jetty.xml.production (hunk 1 above)
Comment (yashrajpchavda) [reply conf/jetty/jetty.xml.production]: done

**Review by silentsakky — APPROVED** [review]

File: conf/jetty/jetty.xml.production [hunk 2]
```diff
@@ -614,7 +614,72 @@
 		    <Set name="replacement">%%zimbraMailURL%%</Set>
 		</New>
 	    </Arg>
-        </Call>       
+        </Call>      
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">/bundle.*</Set>
```
Comment (silentsakky) [inline conf/jetty/jetty.xml.production]: should be `zimbrax/.*/bundle.*`

File: conf/jetty/jetty.xml.production [hunk 3]
```diff
@@ -614,7 +614,72 @@
...
+              <Set name="name">Cache-Control</Set>
+              <Set name="value">Max-Age=86400,public</Set>
+            </New>
+          </Arg>
+        </Call>
+
+        <Call name="addRule">
+          <Arg>
+            <New class="org.eclipse.jetty.rewrite.handler.HeaderPatternRule">
+              <Set name="pattern">.*(.chunk.*)</Set>
```
Comment (silentsakky) [inline conf/jetty/jetty.xml.production]: same as https://github.com/Zimbra/zm-jetty-conf/pull/9/files#r334025445
let's restrict this for zimbrax only

Comment (silentsakky) [reply conf/jetty/jetty.xml.production]: for all below rules

**Pull request merged**